File: fib.java
public class fib {

	public static int rekfib(int n) {
		if(n == 0 || n == 1) {
			return 1;
		}
		return rekfib(n-1)+rekfib(n-2);
	}
	
	public static void main(String[] args){
		System.out.println(rekfib(45));
	}
}

File: fib.c
#include <stdio.h>

int fib(int n) {
	if(n < 2)
		return 1;
	return fib(n-2) + fib(n-1);
}

int main()
{
	printf("%d\n", fib(45));
	return 0;
}
